### ISSUE TITLE
Table as a dependency is injected, but the robot is not

### ISSUE THREAD

**Issue by scepticalcat** [KarthickNarayana/toy_robot#4]
https://github.com/KarthickNarayana/toy_robot/blob/3d570a92d5c8b027e2456ec4f7bce47223719ecf/lib/toy_robot/cli.rb#L5

Injecting the table as a dependency is good, it would have been great to see the robot instance treated similarly.

**Comment by KarthickNarayana** [comment]
Yes.   This way the the boundary is set to the and we change - the boundary independently. I get you when you say Robot should also be injected.
